syntaxes/SQL.plist

```diff
@@ -363,7 +381,13 @@
 		</dict>
 		<dict>
 			<key>match</key>
-			<string>(?i)\b(choose|iif)\b</string>
+			<string>(?i)\b(json|isjson|json_object|json_array|json_value|json_query|json_modify|json_path_exists)\b</string>
+			<key>name</key>
+			<string>support.function.json.sql</string>
+		</dict>
+		<dict>
+			<key>match</key>
+			<string>(?i)\b(choose|iif|greatest|least)\b</string>
```

Comment:
Out of curiosity how did you determine the "sections" each of these were in? Is there some core list you used for that?

Reply:
I grouped based on functions group. For instance all the date time functions are under function_datetime https://docs.microsoft.com/en-us/sql/t-sql/functions/date-bucket-transact-sql?view=sql-server-ver16

I'm actually not sure how to verify them or if my grouping is correct. I just followed the existing pattern

Reply:
Cool, makes sense. Make sure you include that in your documentation!

As for verifying you should look through the VS Code docs and/or talk to Alex to see what we can do - we definitely should verify this all works correctly as early as we can.

Reply:
I verified one of the function name and it showed correct coloring in ADS so I guess the way I did is correct

Reply:
What do you mean? How did you get your changes from the plist file into ADS?

Reply:
format in this file is xml, the one in vscode and ADS is json, that basically the only difference, I can add all these functions there manually and test.

Reply:
Oh cool, that's handy. Would still be worth it to check with Alex to see if there's a way we can generate the full file automatically but at least this is something. 

Did you test with one of the new sections you added as well to make sure they don't need to be added to some other list somewhere?

Reply:
we can write a tool ourself :) convert xml to json. yes I added this 
		{
			"match": "(?i)\\b(asymkey_id|asymkeyproperty|certproperty|cert_id|crypt_gen_random|decryptbyasymkey|decryptbycert|decryptbykey|decryptbykeyautoasymkey|decryptbykeyautocert|decryptbypassphrase|encryptbyasymkey|encryptbycert|encryptbykey|encryptbypassphrase|hashbytes|is_objectsigned|key_guid|key_id|key_name|signbyasymkey|signbycert|symkeyproperty|verifysignedbycert|verifysignedbyasymkey)\\b",
			"name": "support.function.cryptographic.sql"
		},
and tested 
![Uploading Screen Shot 2022-06-17 at 3.20.23 PM.png…]()